packages/block-library/src/pullquote/edit.native.js

```diff
@@ -0,0 +1,140 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import {
+	AlignmentControl,
+	BlockControls,
+	RichText,
+	useBlockProps,
+	getColorObjectByAttributeValues,
+	__experimentalGetColorClassesAndStyles as getColorClassesAndStyles,
+} from '@wordpress/block-editor';
+import { createBlock } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import { Figure } from './figure';
+import { BlockQuote } from './blockquote';
+
+const getBackgroundColor = ( { attributes, colors, style } ) => {
+	const { backgroundColor } = attributes;
+
+	const colorProps = getColorClassesAndStyles( attributes );
+	const colorObject = getColorObjectByAttributeValues(
+		colors,
+		backgroundColor
+	);
+
+	return (
+		colorObject?.color ||
+		colorProps.style?.backgroundColor ||
+		colorProps.style?.background ||
+		style?.backgroundColor
+	);
+};
+
+const getTextColor = ( { attributes, colors, style } ) => {
+	const colorProps = getColorClassesAndStyles( attributes );
+	const colorObject = getColorObjectByAttributeValues(
+		colors,
+		attributes.textColor
+	);
+
+	return colorObject?.color || colorProps.style?.color || style?.color;
+};
+const getBorderColor = ( props ) => {
+	const { wrapperProps, attributes } = props;
+	const { className } = attributes;
+
+	// Checking for the is-style-solid-color we can better approximate the pull quote style
+	const defaultColor =
+		className?.search( 'is-style-solid-color' ) !== -1
+			? getBackgroundColor( props )
+			: getTextColor( props );
+
+	return wrapperProps?.style?.borderColor || defaultColor;
+};
+/**
+ * Internal dependencies
+ */
+
+function PullQuoteEdit( props ) {
+	const { attributes, setAttributes, isSelected, insertBlocksAfter } = props;
+	const { textAlign, citation, value } = attributes;
+
+	const blockProps = useBlockProps( {
+		className: classnames( {
+			[ `has-text-align-${ textAlign }` ]: textAlign,
+		} ),
```

Comment:
We don't use this in native so we can remove it.

Reply:
Good call